staging/cfg/rules.official.json

```diff
@@ -92,124 +42,20 @@
 					"cheater"
 				]
 			},
-			"description": "Poohook",
+			"description": "[TF2BD]Invisible Character/namestealer",
 			"triggers": {
 				"avatar_match": [
 					{
-						"avatar_hash": "fd5b5bae0eeb72930a58702472f2c8c7ace04dfa"
-					},
-					{
-						"avatar_hash": "a6ede29c46bea13b4ba947799815e7336d39cb77"
+						"avatar_hash": ""
 					}
 				],
 				"mode": "match_any",
 				"username_text_match": {
 					"case_sensitive": false,
 					"mode": "contains",
 					"patterns": [
-						"discord.io/poohook"
-					]
-				}
-			}
-		},
-		{
-			"actions": {
-				"mark": [
-					"cheater"
-				]
-			},
-			"description": "[VALVE] Twilight Sparkle",
-			"triggers": {
-				"username_text_match": {
-					"case_sensitive": false,
-					"mode": "contains",
-					"patterns": [
-						"[VALVE] Twilight Sparkle"
-					]
-				}
-			}
-		},
-		{
-			"actions": {
-				"mark": [
-					"cheater"
-				]
-			},
-			"description": "royalhack.net",
-			"triggers": {
-				"username_text_match": {
-					"case_sensitive": false,
-					"mode": "contains",
-					"patterns": [
-						"royalhack.net"
-					]
-				}
-			}
-		},
-		{
-			"actions": {
-				"mark": [
-					"cheater"
-				]
-			},
-			"description": "(catbot) furry-bot",
-			"triggers": {
-				"username_text_match": {
-					"case_sensitive": false,
-					"mode": "regex",
-					"patterns": [
-						"furry-bot \\d+"
-					]
-				}
-			}
-		},
-		{
-			"actions": {
-				"mark": [
-					"suspicious"
-				]
-			},
-			"description": "(catbot) cat.jpg",
-			"triggers": {
-				"username_text_match": {
-					"case_sensitive": true,
-					"mode": "equal",
-					"patterns": [
-						"cat.jpg"
-					]
-				}
-			}
-		},
-		{
-			"actions": {
-				"mark": [
-					"cheater"
-				]
-			},
-			"description": "(catbot) shasta bot",
-			"triggers": {
-				"username_text_match": {
-					"case_sensitive": true,
-					"mode": "starts_with",
-					"patterns": [
-						"shasta bot"
-					]
-				}
-			}
-		},
-		{
-			"actions": {
-				"mark": [
-					"cheater"
-				]
-			},
-			"description": "(catbot) Lag-bot",
-			"triggers": {
-				"username_text_match": {
-					"case_sensitive": false,
-					"mode": "regex",
-					"patterns": [
-						"Lag-Bot \\d+"
+						"\u0e4a",
+						"\u202c"
```

Comment:
add U+200E (Left-To-Right Mark), U+200D (Zero Width Joiner), U+200C (Zero Width Non-Joiner), U+200B (Zero Width Space), U+2060 (Word Joiner), U+202C (Pop Directional Formatting), U+FEFF (Zero Width No-Break Space), and finally the U+200F (Right-To-Left Mark)